ecommerce/api.py

```diff
@@ -218,3 +223,38 @@ def get_new_order_by_reference_number(reference_number):
         return Order.objects.get(id=order_id, status=Order.CREATED)
     except Order.DoesNotExist:
         raise EcommerceException("Order {} is expected to have status 'created'".format(order_id))
+
+
+def enroll_user(order):
```

Comment:
nit: maybe the name should be modified to something that implies that this is given a successful order